src/components/EmailForm.js

```diff
@@ -25,6 +26,7 @@ export default function EmailForm({ className }) {
 
   return (
     <div className={className}>
+      <img className="logo mt-4" src={logo} alt="eden logo" />
       <h3 className="header-text header-text-top mb-5 mt-5">
```

Comment:
Can we remove the mt-5 from here. Thank you.

Reply:
Sure. Just to clarify, do you mean the mt-4 from line 29 (in green)?

Reply:
I mean the mt-5 on line 30.

Reply:
I've done that now.